const/src/quotas.ts

```diff
@@ -46,7 +46,7 @@ export const quotaTypes = [
   'ai_spend',
   'openai_spend',
   'bing_search_spend',
-  'always_alive',
+  'always_alive'
 ] as const satisfies Readonly<string[]>
 
 export const quotaConfigs = {
```

Comment:
In the future this export should be removed to make getter on line 276 the only way to access config objects